File: testprojects/master_scala_mainone/pipelines/ScalaGCP/code/src/main/scala/graph/Reformat_7.scala
package graph

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_7 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.select(col("c_new_str"))

}

File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/pythonorganization/donot/openme12/graph/Reformat_7.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythonorganization.donot.openme12.config.ConfigStore import *
from pythonorganization.donot.openme12.udfs.UDFs import *

def Reformat_7(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        col("customer_id").cast(ShortType()).alias("c_short1"), 
        col("customer_id").cast(IntegerType()).alias("c_int1"), 
        col("customer_id").cast(ShortType()).alias("c_short2"), 
        col("customer_id").cast(IntegerType()).alias("c_int2")
    )